.vscode/settings.json

```diff
@@ -3,7 +3,7 @@
   "editor.formatOnSave": true,
 
   "[typescript]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+    "editor.defaultFormatter": "vscode.typescript-language-features"
```

Comment:
What's the reason for this change?

Reply:
I had issues with Prettier locally, and changed this by mistake. Reverting it.